lightning/src/ln/msgs.rs

```diff
@@ -441,6 +441,53 @@ pub struct ChannelReady {
 	pub short_channel_id_alias: Option<u64>,
 }
 
+/// A splice message to be sent by or received from the splice initiator.
+/// TODO(splicing): Is using 'splice initiator' role OK?
+/// TODO(splicing): Can the channel acceptor later be the splice initiator?
```

Comment:
Yes. The initiator of quiescence (first to send `stfu`) will be the one who must later initiate the splice with `splice`. If both peers send a `stfu` before receiving from the other peer, then the initiator [is always the channel opener](https://github.com/lightning/bolts/blob/a68d7b07cea0fb243f25e955cca6134cae5872cf/02-peer-protocol.md#rationale-4).

Reply:
(This TODO has been removed, looks like a slightly outdated version was reviewed)